prow/clusterissuer.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: cert-manager.io/v1alpha2
```

Comment:
NIT: I wonder if this should live under the `prow/` folder since it's something that we can use to obtain certificate for all ingresses in the cluster. But no reason to hold on this PR for that.